Gemfile

```diff
@@ -14,10 +14,11 @@ gem 'pg', '~> 1.0'
 gem 'webpacker', '~> 4'
 
 # Login via 3rd party services.
-gem 'omniauth', '~> 2.0'
+gem 'omniauth', '~> 1.0'
```

Comment:
Why are we rolling back the omniauth to version 1?

Reply:
omniauth-discourse only seems to work with version 1, but if someone can make it work otherwise I'll change it back.